dash_cytoscape/utils/Tree.py

```diff
@@ -155,3 +173,17 @@ def create_index(self):
                     stack.append(child)
 
         return self.index
+
+
+if __name__ == '__main__':
```

Comment:
Yeah, mostly for users who are curious about how it works, reading the source code will give them an example of running it. Since it's protected by `__name__`, this will only be run when they user runs `python dash_cytoscape/utils/Tree.py`.

Reply:
Do you think I should remove it still?

Reply:
I agree that it's nice for users (newcomers especially) to be able to run `Tree.py` as a standalone script. It is also true that it's not what you'd expect with a module living under a `utils/` subdirectory, from a repo structure, project organization perspective. I would keep it for now, because there's value in getting a quick feel for the trees... We could even make use of `sys.argv`! ;)